docs/shared/packages/webpack/webpack-plugins.md

```diff
@@ -1,9 +1,167 @@
 # Webpack plugins
```

Comment:
Ben asked to put the frontmatter metadata at the top of the page:
```
---
title: This is a custom title
description: This is a custom description
---
```

Reply:
Updated